### PR TITLE
AuthButton 아이콘 수정

### PR DESCRIPTION
**Pull request by chepchep2**

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#33** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#32** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#31** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/31?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#30** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#29** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/29?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#28** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/28?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#27** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#26** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/25?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#23** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/23?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#22** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/22?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#21** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#20** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/20?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#19** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/19?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#18** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/18?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/page/intro_page.dart [hunk 1]
```diff
@@ -50,23 +50,28 @@ class IntroPage extends StatelessWidget {
                 const SizedBox(height: 16),
                 AuthButton(
                   onClick: () {},
-                  icon: const FaIcon(FontAwesomeIcons.k),
+                  icon: Image.asset('assets/icon/카카오 로고.png'),
```
Comment (flowkater) [inline lib/page/intro_page.dart:53]: 파일 이름은 전부 영어로 관리해주세요.

File: lib/page/intro_page.dart [hunk 2]
```diff
@@ -50,23 +50,28 @@ class IntroPage extends StatelessWidget {
...
                   text: '카카오로 계속하기',
                 ),
                 const SizedBox(height: 16),
                 AuthButton(
-                    onClick: () {},
-                    icon: const FaIcon(FontAwesomeIcons.n),
-                    backgroundColor: AppColor.greenColor,
-                    text: '네이버로 계속하기'),
+                  onClick: () {},
+                  icon: Image.asset('assets/icon/네이버 로고.png'),
```
Comment (flowkater) [inline lib/page/intro_page.dart:60]: 파일 이름 영어로 수정

File: lib/page/intro_page.dart [hunk 3]
```diff
@@ -50,23 +50,28 @@ class IntroPage extends StatelessWidget {
...
+                  backgroundColor: AppColor.greenColor,
+                  text: '네이버로 계속하기',
```
Comment (flowkater) [inline lib/page/intro_page.dart:62]: 앱에 들어간 모든 텍스트도 전부 상수로 빼서 관리하면 좋을 것 같습니다.
(추후에 번역 들어가는거 생각하면)

**Review by flowkater — CHANGES_REQUESTED** [review]
파일 이름 영어 체크

**Comment by chepchep2** [comment]
### Merge activity

* **Mar 17, 8:12 PM EDT**: @chepchep2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21).
* **Mar 17, 8:20 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21) rebased this pull request as part of a merge.
* **Mar 17, 8:20 PM EDT**: @chepchep2 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21).

**Pull request merged**